changelog/changelog.go

```diff
@@ -5,28 +5,28 @@ import (
 )
 
 const (
-	tplStr = `{{.OldRelease}} -> {{.NewRelease}}
+	tplStr = `### {{.OldRelease}} -> {{.NewRelease}}
 
 {{ if (len .Features) gt 0 }}
-# Features
+##### Features
```

Comment:
looks like `Features` gets a slightly smaller header size than the rest (one more `#`) -- intentional?